try {
} catch (err) {
    try {
    } catch (err) {
        throw err
    }
}
